Tasky/TaskyAndroid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="2.0" package="com.xamarin.samples.taskyandroid" android:versionCode="2">
 	<application android:label="Tasky" android:debuggable="true" android:icon="@drawable/icon" android:theme="@style/Theme.Tasky"></application>
-	<uses-sdk android:minSdkVersion="15" />
+	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="25" />
```

Comment:
i think 15 is fine for the minimum

Reply:
Unfortunately calabash and XamarinUITests support android api 8+, so we need tasky app for all supported devices

Reply:
soon (August) the minimum will be 26 ([reference](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html)). 

The value was previously `"15"` and should be left at 15 - the only change required to address the popup warning is the `targetSdkVersion`.